code/frameworks/angular/src/server/framework-preset-angular-ivy.ts

```diff
@@ -58,7 +58,9 @@ export const webpack = async (webpackConfig: Configuration, options: PresetOptio
     return webpackConfig;
   }
 
-  runNgcc();
+  if(angularOptions.enableNgcc !== false) {
```

Comment:
Wouldn't `!== false` trigger for `undefined` or `null` as well? If `enableNgcc` is not defined at all, then this is true, too 

Probably safer to check for `=== true` or `!!angularOptions.enableNgcc`?

Just by looking at the changes, I'm not even sure if `angularOptions` is guaranteed (I'm on vacation, sorry 🙂 can you check this for me?)

If not, then `if( angularOptions?.enableNgcc )` is probably the safest way

Reply:
Actually, I really seem to be on vacation-mode

The default value is "true" so if it is null or undefined, it should also be true. Ignore my comment 🙃